amazonia/defaults.yaml

```diff
@@ -36,6 +36,7 @@ iam_instance_profile_arn:
 zd_autoscaling_units:
 autoscaling_units:
 database_units:
+ec2_scheduled_shutdown:
```

Comment:
Yeah it's more explicit to set it to False.

Right now, it has the same behaviour (None evaluates to False for conditional statements), but it would be more obvious to just have it default to False.